src/sheet/components/SheetDataEditor.tsx

```diff
@@ -149,17 +150,21 @@ export default function SheetDataEditor({
   return (
     <div>
       <div className="my-5 flex items-center">
-        <div>
+        <div className="mr-8">
           <ButtonGroup activeButton={viewMode} buttons={viewModeButtons} />
         </div>
 
-        {/* TODO: Add tooltip when disabled */}
-        <TrashIcon
-          className={`ml-8 h-6 w-6 ${
-            selectedRows.length > 0 ? 'cursor-pointer' : disabledButtonClasses
-          }`}
-          onClick={() => setRemoveConfirmationModalOpen(true)}
-        />
+        <div className="group relative">
```

Comment:
Shouldn't this be inside the tooltip component and we would pass TrashIcon as a child of it?